commands/help.rb

```diff
@@ -101,16 +101,16 @@ def self.help(command)
       puts <<~EOT
         Look for package(s).
         Usage: crew search [-v|--verbose] [<pattern> ...]
-        If <pattern> is omitted, all packages will be returned.
+        Both the name and description of packages will be searched.
         If the package color is " + "green".lightgreen + ", it means the package is installed.
         If the package color is " + "red".lightred + ", it means the architecture is not supported.
+        If the package color is " + "blue".lightblue + ", it means the architecture is supported but the package is not installed.
         The <pattern> string can also contain regular expressions.
-        If `-v` or `--verbose` is present, homepage, version and license will be displayed.
+        If `-v` or `--verbose` is present, the homepage, version and license of found packages will be displayed.
         Examples:
-          crew search ^lib".lightblue + " will display all packages that start with `lib`.
-          crew search audio".lightblue + " will display all packages with `audio` in the name.
-          crew search | grep -i audio".lightblue + " will display all packages with `audio` in the name or description.
```

Comment:
Why was this line removed?  It's definitely a good example.

Reply:
The same functionality can now be achieved with `crew search audio`.